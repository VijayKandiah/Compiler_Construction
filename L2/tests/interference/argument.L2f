(:storeValueToArray
  2 0
  %A1 <- rdi
  %a <- rsi
  %length <- mem %A1 0 
  return
)
