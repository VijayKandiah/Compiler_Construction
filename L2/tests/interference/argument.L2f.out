A1 a r12 r13 r14 r15 rax rbp rbx rsi
a A1 r12 r13 r14 r15 rax rbp rbx
length r12 r13 r14 r15 rax rbp rbx
r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi
r11 r10 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi
r12 a A1 length r10 r11 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi
r13 a A1 length r10 r11 r12 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi
r14 a A1 length r10 r11 r12 r13 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi
r15 a A1 length r10 r11 r12 r13 r14 r8 r9 rax rbp rbx rcx rdi rdx rsi
r8 r10 r11 r12 r13 r14 r15 r9 rax rbp rbx rcx rdi rdx rsi
r9 r10 r11 r12 r13 r14 r15 r8 rax rbp rbx rcx rdi rdx rsi
rax a A1 length r10 r11 r12 r13 r14 r15 r8 r9 rbp rbx rcx rdi rdx rsi
rbp a A1 length r10 r11 r12 r13 r14 r15 r8 r9 rax rbx rcx rdi rdx rsi
rbx a A1 length r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rcx rdi rdx rsi
rcx r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rdi rdx rsi
rdi r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdx rsi
rdx r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rsi
rsi A1 r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx
